builder/azure/arm/config.go

```diff
@@ -109,6 +109,9 @@ type Config struct {
 
 	Comm communicator.Config `mapstructure:",squash"`
 	ctx  *interpolate.Context
+
+	// SAS URL lifetime in months
+	SasUrlLifetime int `mapstructure:"sas_url_lifetime"`
```

Comment:
This should be a duration.  It gives much better granularity, and is more intuitive.